internal/core/core.go

```diff
@@ -402,13 +415,45 @@ func (v VDSHandle) GetMetadata() ([]byte, error) {
 	return buf, nil
 }
 
-func (v VDSHandle) GetSlice(lineno, direction int) ([]byte, error) {
+func newCSliceBounds(bounds []Bound) ([]C.struct_Bound, error) {
+	var cBounds []C.struct_Bound
+	for _, bound := range bounds {
+		direction, err := GetAxis(bound.Direction)
+		if err != nil {
+			return nil, err
+		}
+
+		cBound := C.struct_Bound{
+			C.float(bound.Lower),
+			C.float(bound.Upper),
+			C.enum_axis_name(direction),
```

Comment:
Because GetAxis is a public function that is called from `endpoint.go`. This core module is supposed to hide all the C crap from the rest of the go code base. As it's not nice to leak CGO around everywhere